### PR TITLE
[sass] Add new package

### PR DESCRIPTION
**Pull request by vincowl**
Ruby' sass gem is now obsolete and has been migrated to dart-sass. This package solves this

### CODE REVIEW

File: packages/sass.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  depends_on 'dart'
+
+  def self.install
+    system 'pub get'
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/dart-sass"
+    FileUtils.cp_r '.', "#{CREW_DEST_PREFIX}/share/dart-sass"
+    system "echo '#!/bin/bash' >> sass" 
+    system "echo 'dart #{CREW_PREFIX}/share/dart-sass/bin/sass.dart $@' >> sass"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
+    FileUtils.cp_r 'sass', "#{CREW_DEST_PREFIX}/bin/."
```
Comment (uberhacker) [inline packages/sass.rb:25]: Better use `system "install -Dm755 'sass', #{CREW_DEST_PREFIX}/bin/sass"` here instead to make the script executable.  You also can remove line 24 above.

File: packages/sass.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  })
+
+  depends_on 'dart'
+
+  def self.install
+    system 'pub get'
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share/dart-sass"
+    FileUtils.cp_r '.', "#{CREW_DEST_PREFIX}/share/dart-sass"
+    system "echo '#!/bin/bash' >> sass" 
+    system "echo 'dart #{CREW_PREFIX}/share/dart-sass/bin/sass.dart $@' >> sass"
```
Comment (uberhacker) [inline packages/sass.rb:23]: This could be a build step.

Comment (vincowl) [reply packages/sass.rb:23]: If I put lines 22/23 in a build step and keep the new install line (equivalent to lines 24-26) in an install step, won't we face troubles once binaries will be produced and we then won't use the build step ? install command will not find the sass script.

Putting all this in a build step doesn't work. After install sass is not found in /usr/local/bin

File: packages/sass.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  binary_url ({
+  })
+  binary_sha256 ({
+  })
+
+  depends_on 'dart'
+
+  def self.build
+    system "echo '#!/bin/bash' >> sass" 
+    system "echo 'dart #{CREW_PREFIX}/share/dart-sass/bin/sass.dart $@' >> sass"
```
Comment (uberhacker) [inline packages/sass.rb:20]: Does the `$` need escaped?  What are the contents of sass after install?

Comment (vincowl) [reply packages/sass.rb:20]: After installing, here is the content of `sass`. Seems it doesn't need to escape $
```
$ cat /usr/local/bin/sass 
#!/bin/bash
dart /usr/local/share/dart-sass/bin/sass.dart $@
```

**Pull request merged**